YOLONAS-POSE.md

```diff
@@ -0,0 +1,110 @@
+# YOLO-NAS-POSE
+### A Next-Generation, Pose Estimation Foundational Model generated by Deci’s Neural Architecture Search Technology
+
+Deci is thrilled to announce the release of a new object detection model, YOLO-NAS-POSE - a successor of [YOLO-NAS](YOLONAS.md), 
+pose estimation architecture, providing superior real-time object detection capabilities and production-ready performance. 
+Deci's mission is to provide AI teams with tools to remove development barriers and attain efficient inference performance more quickly.
+
+![YOLO-NAS-POSE](documentation/source/images/yolo_nas_pose_frontier_t4.png)
+
+The new YOLO-NAS-POSE delivers state-of-the-art (SOTA) performance with the unparalleled accuracy-speed performance, outperforming other models such as YOLOv8-Pose, DEKR and others.
+
+Deci's proprietary Neural Architecture Search technology, [AutoNAC™](https://deci.ai/technology/), generated the architecture of YOLO-NAS-POSE model. 
+The AutoNAC™ engine lets you input any task, data characteristics (access to data is not required), inference environment and performance targets, 
+and then guides you to find the optimal architecture that delivers the best balance between accuracy and inference speed for your specific application. 
+In addition to being data and hardware aware, the AutoNAC engine considers other components in the inference stack, including compilers and quantization.
+
+| Model            | AP    | Latency (ms) |
+|------------------|-------|--------------|
+| YOLO-NAS N       | 59.68 | 2.35 ms      |
+| YOLO-NAS S       | 64.15 | 3.29 ms      |
+| YOLO-NAS M       | 67.87 | 6.87 ms      |
+| YOLO-NAS L       | 68.24 | 8.86 ms      |
+
+AP numbers in table reported for COCO 2017 Val dataset and latency benchmarked for 640x640 images on Nvidia T4 GPU.
+No flip-TTA was used.
+
+Similarly to YOLO-NAS, YOLO-NAS-POSE architecture employs quantization-aware blocks and selective quantization for optimized performance. 
+In fact YOLO-NAS-POSE is a direct successor of YOLO-NAS and uses same backbone and neck as YOLO-NAS. 
```

Comment:
successor here again

Reply:
Fixed in f4bc27730ea7141b7f07ea493d68bc8e61078776